changelog.py

```diff
@@ -0,0 +1,168 @@
+import gitlab
+import os.path
+import re
+import semver
+import yaml
+
+from dateutil import parser
+from datetime import datetime, timedelta, timezone
+from jinja2 import Template
+from typing import Dict
+from urllib.parse import unquote
+
+
+path = os.path.join(os.path.dirname(__file__), "template.yml")
+with open(path) as f:
+    action = yaml.safe_load(f)
+DEFAULT_TEMPLATE = action["changelog"]["default"]
+
+DEFAULT_IGNORE = [
+    "changelog skip",
+    "duplicate",
+    "exclude from changelog",
+    "invalid",
+    "no changelog",
+    "question",
+    "wont fix",
+]
+
+
+class Changelog:
+    """A Changelog produces release notes for a single release."""
+
+    def __init__(self, repo):
+        self._repo = repo
+        self._template = Template(DEFAULT_TEMPLATE, trim_blocks=True)
```

Comment:
If you're not taking in a template argument does it make sense to read in the default template at a global level? Could just read it in here I think. 
Though I guess you're going for a `const` feel for it. Up to you, but seems a little weird to me to read in the file just by importing the module. 
Perhaps the const could just be like `DEFAULT_TEMPLATE_PATH` which you can then read in the init?

Reply:
Or maybe use a class variable and read it into the variable only the first time an instance is initiated, if you're always using the same template and don't want to read on every init.